src/libraries/System.Private.CoreLib/src/System/Threading/ThreadPool.Portable.cs

```diff
@@ -31,6 +31,9 @@ public static partial class ThreadPool
             AppContextConfigHelper.GetBooleanConfig("System.Threading.ThreadPool.EnableWorkerTracking", false);
 #endif
 
+        internal static bool EnableDispatchAutoreleasePool { get; } =
```

Comment:
This is a `partial class`. Isn't there a file where you can put this property so it doesn't need to be duplicated?

Reply:
There actually isn't. `ThreadPool.cs` didn't actually have the `ThreadPool` type and I don't want to make a new `ThreadPool.cs` in the same PR where I'm renaming the old one because once it's squashed, the git history will get all messed up.

Reply:
Can we put it in `ThreadPoolWorkQueue.AutoreleasePool.OSX.cs`?

```
public static partial class ThreadPool
{
    internal static bool EnableDispatchAutoreleasePool { get; } = AppContextConfigHelper.GetBooleanConfig("System.Threading.ThreadPool.EnableDispatchAutoreleasePool", false);
}
```
I understand that it is a minor violation of coding conventions, but I think it is acceptable.

Also, we won't be unnecessarily reading the property on non-OSX platforms anymore.

Reply:
Done!